product-development/README.md

```diff
@@ -104,57 +117,99 @@ Example:
+
+It doesn't matter what form this backlog takes. It can be a shared Google Doc, or a
+clubhouse story, or a piece of paper. You can think of this as ranked-choice
+voting for what you want to work on.
+
+Every two weeks, Nick will talk to you about your list, and try to get a sense
+of what you think is important and want to work on.
 
-<img src="images/above-the-line.png" height="150" />
+Nick will assign epics in the backlog, trying to balance preferences and current
+priority order.
 
-## Reducing scope
-A given epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 business days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 business days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work. Alternatively, the engineer(s) may create a separate epic with the smaller scope and start work on that one instead (putting the original epic back into the backlog).
+If there's an epic that's high in priority order but no one wants to work on,
+then Nick should dig more into why. Do we need to do more research upfront?  Is
+the epic defined incorrectly? Is it misaligned with what we're currently
+focusing on?
+
+There may be rare cases where someone gets assigned to an epic that they
+don't want to work on, but in those cases we should explicitly call out
+that this is happening and why.
+
+At all times, every Tilter should have 1-2 epics assigned to them that haven't
+been started yet. The epics are their "Next Up" list.  This isn't final - they
+may get reassigned. But we bias towards keeping the current assignment so that
+Tilters know what's coming next.
+
+### Starting an Epic
+
+When you're ready to start an epic, it should be clear which epic to start work
+on -- it's the highest-priority backlog epic assigned to you.
+
+Move it into the `In Progress` state.
+
+## How an Epic Progresses
+
+A given epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 business days_** (counted from when the epic moves into `In Progress`, more below on states). 
+
+### Reducing Scope
+
+If the solution cannot be finished within 7 business days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work. Alternatively, the engineer(s) may create a separate epic with the smaller scope and start work on that one instead (putting the original epic back into the backlog).
```

Comment:
Out of curiosity: does anyone need to sign off on these epic changes (scope reduction etc.)?

Reply:
@maiamcc : This isn't a new section. (Just Git/GitHub can't handle Nick moving around things properly and it looks like it's new in this diff.) 

I don't think we've had any formal scope sign-off process (whether initial or changes to scope). I don't think Nick is suggesting a sign-off process here. But I'm sure he'll comment if this isn't accurate.

Reply:
hehe, ya, i do think it deserves more color here on what's actually happening. my vague impression is that everyone has done a good job of reducing scope when necessary, and communicating when the scope is being reduced in radical ways

Reply:
added some more color here!